docs/image.rst

```diff
@@ -39,8 +51,17 @@ Classes
 
     The other form creates an empty image with ``width`` columns and
     ``height`` rows. Optionally ``buffer`` can be an array of
-    ``width``×``height`` integers in range 0-9 to initialize the image.
-
+    ``width``×``height`` integers in range 0-9 to initialize the image:
+   
+        Image(2, 2, b'\x08\x08\x08\x08')
+    or
```

Comment:
Without the additional lines the formatting goes a bit wonky:

![image](https://user-images.githubusercontent.com/29712657/47505898-f0062b00-d866-11e8-9f39-74e7c4632cb2.png)


```suggestion

    or::

```